cmd/coder/urls.go

```diff
@@ -29,7 +141,9 @@ func (cmd urlsCmd) Spec() cli.CommandSpec {
 	}
 }
 
-func (cmd urlsCmd) Run(fl *pflag.FlagSet) {
+// urlList() returns the list of active devURLs from the cemanager server.
+
+func urlList(fl *pflag.FlagSet) []DevURL {
```

Comment:
Accepting a flag set makes it difficult to understand what this function actually requires. I'd suggest passing `envName` in instead.

Reply:
Ah, yes I might not need coupling to the entire FlagSet